### ISSUE TITLE
TouchSizeWCAG- wrong calculation when element is partially visible

### ISSUE THREAD

**Issue by ferBonnin** [dequelabs/axe-android#53]
When an element is only partially visible in the screen, there is a false-positive with TouchSizeWcag

Found in: Sunfloweralpha Android demo app. 
Go to Plant list and scroll until one of the cards is only partially visible.

### Screenshot

Attach a Screenshot with of the violation highlighted using Axe for Android Google Play App.
![sunflower app, cilantro card partially visible and failing](https://user-images.githubusercontent.com/9062104/72569518-7db8a680-386f-11ea-81e3-459f233734de.png)

### Why the Control is Accessible

The element is partially visible.
If the element gets moved and fully visible, then its no longer highlighted as a failure

### Version Information

 - Device: [e.g. Moto G6] Nokia 7.1
 - OS Version: [e.g. 7.1] Android 9

### Additional information
Might be related to #30 
The Accessibility Insights team is investigating more on this :)

**Comment by chriscm2006** [comment]
Thanks Fer, we'll look into this!

**Comment by chriscm2006** [comment]
We expect this fix to be released here soon and to the App Store within the month.

**Comment by devchan4188** [comment]
This issue has been resolved, the fix will be available in 0.1.17

**Comment by ferBonnin** [comment]
awesome! @devchan4188 is there an ETA on when will 0.1.17 will be released? thanks!
